src/CONST.ts

```diff
@@ -43,6 +43,7 @@ const CONST = {
         OUT: 'out',
     },
     ARROW_HIDE_DELAY: 3000,
+    INPUT_FOCUS_DELAY: 600,
```

Comment:
How was this time arrived? @akinwale

Reply:
@abdulrahuman5196 During my testing on native iOS and Android, it turns out that the 300ms delay was not enough for the input ref to be initialised, so it would still be null at the point where focus is being set. I also tested with 400ms and 500ms which gave the same behaviour, before I finally settled at 600ms.

Reply:
@akinwale 600 is little high IMO like the input focusing will only happen 300 after the `ANIMATED_TRANSITION(300)` is complete. Which could look weird. 

What is the lowest value we are able to set? Is anything below 600 not working?

Reply:
@abdulrahuman5196 On native, nothing below 600 works reliably. 500 works occasionally, but it's not 100% of the time.

Reply:
Got it. I think it specific to this particular page. Other pages with the same logic `ANIMATED_TRANSITION` is doing the job.


then for default value lets keep `ANIMATED_TRANSITION` and pass from the component the delay as 600.

@akinwale

Reply:
@akinwale This is still not fixed.

Reply:
I think this is missed

Reply:
@abdulrahuman5196 Can you explain? We still need to use this value in the `WorkspaceNewRoomPage` component. I have updated the default value for the `delay` parameter in the hook to be `CONST.ANIMATED_TRANSITION`.

Reply:
OOPS. Sorry I didn't see that. 
Can we name this variable alone in some component specific name instead of `INPUT_FOCUS_DELAY` which is generic? Others shouldn't end up mistaking like me 😮‍💨 
@akinwale